packages/create-block-tutorial-template/index.js

```diff
@@ -5,35 +5,36 @@ const { join } = require( 'path' );
 
 module.exports = {
 	defaultValues: {
-		slug: 'gutenpride',
-		category: 'text',
-		title: 'Gutenpride',
-		description:
-			'A Gutenberg block to show your pride! This block enables you to type text and style it with the color font Gilbert from Type with Pride.',
-		dashicon: 'flag',
+		slug: 'copyright-date-block',
+		title: 'Copyright Date',
+		description: "Display your site's copyright date.",
+		npmDependencies: [ '@wordpress/icons' ],
```

Comment:
What would be the reason for installing this package as the dependency? Ah, it isn't exposed as an entry point so it needs to get bundled during the development process. That isn't the best experience in the learning process.

Reply:
Yeah, that's a good point. It is not intuitive that you need to bundle the icons package. I could show how to add a custom SVG icon (which would be the same icon that's currently used). This would not require the icons package and would be a useful demonstration. Something like:

```
const calendarIcon = (
	<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false">
		<path d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm.5 16c0 .3-.2.5-.5.5H5c-.3 0-.5-.2-.5-.5V7h15v12zM9 10H7v2h2v-2zm0 4H7v2h2v-2zm4-4h-2v2h2v-2zm4 0h-2v2h2v-2zm-4 4h-2v2h2v-2zm4 0h-2v2h2v-2z"></path>
	</svg>
);
```

Reply:
I dream of a system where you can simply provide the string with the predefined name of the icon and the rest happens automatically. It worked like that with dashicons, but it had a limited number of options.

Anyway, your idea of using a svg might be the simplest approach or:
- https://developer.wordpress.org/resource/dashicons/#calendar
- https://developer.wordpress.org/resource/dashicons/#calendar-alt

Reply:
> I dream of a system where you can simply provide the string with the predefined name of the icon and the rest happens automatically.

Yeah I agree 💯. I am hesitant to use Dashicons, though, since they do not match the Editor aesthetic. I am on a mission to get custom blocks to look as much like Core blocks as possible 😅 

I will update this PR to use a custom icon. In a way, it actually makes the icon more intuitive from a new developer's perspective.

Reply:
I added a custom icon and removed the dependency in the latest commit.